discovery-provider/default_config.ini

```diff
@@ -3,12 +3,12 @@ start_block = 0x0
 loglevel_flask = DEBUG
 ; do not configure the log level here as this gets overridden by celery lib during setup
 ; set log level via command line in docker yml files instead
-; loglevel_celery = INFO 
+; loglevel_celery = INFO
 block_processing_window = 20
 blacklist_block_processing_window = 600
 peer_refresh_interval = 3000
 identity_service_url = https://identityservice.test
-user_metadata_service_url = '' 
+user_metadata_service_url = http://cn1_creator-node_1:4000
```

Comment:
just curious, what's this URL guy?

Reply:
@piazzatron is this something we can template in like the identity_service_url and avoid committing a local only default config? identityservice.test is an invalid URL by design to force us to pass a config in for prod